README.md

```diff
@@ -62,42 +71,81 @@ We will also keep track of the actions that actually took place and readings tak
 | name       | `String`      |
 | sensor_id  | `Foreign Key` |
 | value      | `Float`       |
-| created_at | `Datetime`   |
+| created_at | `Datetime`    |
 
 ### Scheduled Actions
 
 | Name       | Type          |
 | ---------- | ------------- |
 | id         | `Key`         |
 | name       | `String`      |
```

Comment:
we don't need this name either. We should also change the actuator field to actuator_id